### PR TITLE
Prepare docs for new providers release (August 2022)

### PR DESCRIPTION
**Pull request by potiuk**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by potiuk** [comment]
cc: @vincbeck @ferruzzi  @alexott @wojsamjan -> please take a look and see if there is anything else we should include

**Comment by alexott** [comment]
lgtm for Databricks side, only maybe we don't need this one: `Databricks: update user-agent string (#25578)` - it's covered by the improving telemetry one

**Comment by potiuk** [comment]
> lgtm for Databricks side, only maybe we don't need this one: `Databricks: update user-agent string (#25578)` - it's covered by the improving telemetry one

I prefer to keep full list of the "code" changes. that have potential to change behaviours of released code (As opposed to changes that just impact test code/documentation which can be "hidden") - having the PR# in changelog is  good way to find out problem reasons when they happen

**Comment by potiuk** [comment]
And mostly they make communication with users easier. Changelog is essenially a "communication" tool with the users - it is easier to explain to user "this change was done, impacted this  and actually you can see it there - it was mentioned there in the changelog"

**Comment by alexott** [comment]
ok

**Comment by potiuk** [comment]
cc: @lwyszomi too

**Comment by potiuk** [comment]
Yep. Just approved and merged it.

**Comment by ferruzzi** [comment]
I was hoping to get https://github.com/apache/airflow/pull/25413 in this one since it'll be a major release for Amazon.

**Comment by eladkal** [comment]
> I was hoping to get #25413 in this one since it'll be a major release for Amazon.

I'm not sure how this one will be a breaking change?
It should be backward compatible.

**Comment by ferruzzi** [comment]
It's also removing deprecated code, so I figured this was a good time.  I just pushed a change that is passing all local testing, so we'll see.  No need to hold up the release if you don't think it's important to get it in before the major.

**Comment by eladkal** [comment]
https://github.com/apache/airflow/pull/18896 is probably also in the waiting list for next wave?

**Comment by potiuk** [comment]
> #18896 is probably also in the waiting list for next wave?

Merged.

File: airflow/providers/google/CHANGELOG.rst [hunk 1]
```diff
@@ -23,6 +23,47 @@
...
 
+8.3.0
+.....
+
+Features
+~~~~~~~~
+
+* ``add description method in BigQueryCursor class (#25366)``
+* ``Add project_id as a templated variable in two BQ operators (#24768)``
+* ``Remove deprecated modules in Amazon provider (#25543)``
```
Comment (raphaelauv) [inline airflow/providers/google/CHANGELOG.rst:34]: amazon commit in gcp provider ?

Comment (potiuk) [reply airflow/providers/google/CHANGELOG.rst:34]: Yep. some (typing) changes in Google Provider were done as part of the Amazon change (see yoursefl).

**Comment by potiuk** [comment]
Updated, and reeady for merging when green.

**Comment by potiuk** [comment]
I need +1 here :)

File: airflow/providers/amazon/CHANGELOG.rst [hunk 2]
```diff
@@ -24,6 +24,55 @@
...
+.....
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+* ``Avoid requirement that AWS Secret Manager JSON values be urlencoded. (#25432)``
+* ``Remove deprecated modules (#25543)``
+* ``Resolve Amazon Hook's 'region_name' and 'config' in wrapper (#25336)``
+* ``Resolve and validate AWS Connection parameters in wrapper (#25256)``
+* ``Standardize AwsLambda (#25100)``
```
Comment (eladkal) [inline airflow/providers/amazon/CHANGELOG.rst:37]: `AwsLambda` is not a breaking change https://github.com/apache/airflow/pull/25100
It's backward compatible and the deprecation of this class/modules were not removed in Vincent PRs
https://github.com/apache/airflow/pull/25543#discussion_r938497233 , https://github.com/apache/airflow/pull/25609#discussion_r941209559

**Review by eladkal — APPROVED** [review]

**Pull request merged**